---
title: Analysis of ISCB honorees and keynotes reveals disparities
keywords:
- computational biology
- iscb
- disparities
lang: en-US
date-meta: '2021-03-29'
author-meta:
- Trang T. Le
- Daniel S. Himmelstein
- Ariel A. Hippen
- Matthew R. Gazzara
- Casey S. Greene
manubot-fail-on-errors: true
header-includes: |-
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="Analysis of ISCB honorees and keynotes reveals disparities" />
  <meta name="citation_title" content="Analysis of ISCB honorees and keynotes reveals disparities" />
  <meta property="og:title" content="Analysis of ISCB honorees and keynotes reveals disparities" />
  <meta property="twitter:title" content="Analysis of ISCB honorees and keynotes reveals disparities" />
  <meta name="dc.date" content="2021-03-29" />
  <meta name="citation_publication_date" content="2021-03-29" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Trang T. Le" />
  <meta name="citation_author_institution" content="Department of Biostatistics, Epidemiology and Informatics, Institute for Biomedical Informatics, University of Pennsylvania" />
  <meta name="citation_author_orcid" content="0000-0003-3737-6565" />
  <meta name="twitter:creator" content="@trang1618" />
  <meta name="citation_author" content="Daniel S. Himmelstein" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania" />
  <meta name="citation_author_orcid" content="0000-0002-3012-7446" />
  <meta name="twitter:creator" content="@dhimmel" />
  <meta name="citation_author" content="Ariel A. Hippen" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania" />
  <meta name="citation_author_orcid" content="0000-0001-9336-6543" />
  <meta name="twitter:creator" content="@ariel_hippen" />
  <meta name="citation_author" content="Matthew R. Gazzara" />
  <meta name="citation_author_institution" content="Department of Genetics, Perelman School of Medicine, University of Pennsylvania" />
  <meta name="citation_author_orcid" content="0000-0001-7710-4551" />
  <meta name="twitter:creator" content="@MR_Gazzara" />
  <meta name="citation_author" content="Casey S. Greene" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation" />
  <meta name="citation_author_orcid" content="0000-0001-8713-9213" />
  <meta name="twitter:creator" content="@greenescientist" />
  <link rel="canonical" href="https://greenelab.github.io/iscb-diversity-manuscript/" />
  <meta property="og:url" content="https://greenelab.github.io/iscb-diversity-manuscript/" />
  <meta property="twitter:url" content="https://greenelab.github.io/iscb-diversity-manuscript/" />
  <meta name="citation_fulltext_html_url" content="https://greenelab.github.io/iscb-diversity-manuscript/" />
  <meta name="citation_pdf_url" content="https://greenelab.github.io/iscb-diversity-manuscript/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://greenelab.github.io/iscb-diversity-manuscript/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://greenelab.github.io/iscb-diversity-manuscript/v/94fc5c9ee806b726c0280611c9b554d6872bb29b/" />
  <meta name="manubot_html_url_versioned" content="https://greenelab.github.io/iscb-diversity-manuscript/v/94fc5c9ee806b726c0280611c9b554d6872bb29b/" />
  <meta name="manubot_pdf_url_versioned" content="https://greenelab.github.io/iscb-diversity-manuscript/v/94fc5c9ee806b726c0280611c9b554d6872bb29b/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <meta property="og:image" content="https://github.com/greenelab/iscb-diversity-manuscript/raw/94fc5c9ee806b726c0280611c9b554d6872bb29b/build/assets/thumbnail.png" />
  <meta property="twitter:image" content="https://github.com/greenelab/iscb-diversity-manuscript/raw/94fc5c9ee806b726c0280611c9b554d6872bb29b/build/assets/thumbnail.png" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
bibliography:
- content/manual-references.json
manubot-output-bibliography: output/references.json
manubot-output-citekeys: output/citations.tsv
manubot-requests-cache-path: ci/cache/requests-cache
manubot-clear-requests-cache: false
...



_A DOI-citable version of this manuscript is available at <https://doi.org/10.1101/2020.04.14.927251>_.

This version of the manuscript [contains changes](https://github.com/greenelab/iscb-diversity-manuscript/compare/v1.0...94fc5c9ee806b726c0280611c9b554d6872bb29b) subsequent to the [version 1.0 release](https://github.com/greenelab/iscb-diversity-manuscript/releases/tag/v1.0).


<small><em>
This manuscript
([permalink](https://greenelab.github.io/iscb-diversity-manuscript/v/94fc5c9ee806b726c0280611c9b554d6872bb29b/))
was automatically generated
from [greenelab/iscb-diversity-manuscript@94fc5c9](https://github.com/greenelab/iscb-diversity-manuscript/tree/94fc5c9ee806b726c0280611c9b554d6872bb29b)
on March 29, 2021.
</em></small>

<!-- include the Font Awesome library, per: https://fontawesome.com/start -->
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css">

## Authors



+ **Trang T. Le**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-3737-6565](https://orcid.org/0000-0003-3737-6565)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [trang1618](https://github.com/trang1618)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [trang1618](https://twitter.com/trang1618)<br>
  <small>
     Department of Biostatistics, Epidemiology and Informatics, Institute for Biomedical Informatics, University of Pennsylvania
  </small>

+ **Daniel S. Himmelstein**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-3012-7446](https://orcid.org/0000-0002-3012-7446)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [dhimmel](https://github.com/dhimmel)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [dhimmel](https://twitter.com/dhimmel)<br>
  <small>
     Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania
     · Funded by the Gordon and Betty Moore Foundation ([GBMF4552](https://www.moore.org/grant-detail?grantId=GBMF4552))
  </small>

+ **Ariel A. Hippen**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-9336-6543](https://orcid.org/0000-0001-9336-6543)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [arielah](https://github.com/arielah)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [ariel_hippen](https://twitter.com/ariel_hippen)<br>
  <small>
     Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania
  </small>

+ **Matthew R. Gazzara**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-7710-4551](https://orcid.org/0000-0001-7710-4551)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [mrgazzara](https://github.com/mrgazzara)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [MR_Gazzara](https://twitter.com/MR_Gazzara)<br>
  <small>
     Department of Genetics, Perelman School of Medicine, University of Pennsylvania
  </small>

+ **Casey S. Greene**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-8713-9213](https://orcid.org/0000-0001-8713-9213)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [cgreene](https://github.com/cgreene)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [greenescientist](https://twitter.com/greenescientist)<br>
  <small>
     Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania; Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation
     · Funded by the Gordon and Betty Moore Foundation ([GBMF4552](https://www.moore.org/grant-detail?grantId=GBMF4552))
  </small>



## Abstract

Delivering a keynote talk at a conference organized by a scientific society, or being named as a fellow by such a society, indicates that a scientist is held in high regard by their colleagues.
To explore if the distribution of such indicators of esteem in the field of bioinformatics reflects the composition of this field,
we compared the gender, name-origin and country of affiliation of 412 researchers who had been recognized by the _International Society for Computational Biology_ (75 fellows and 337 keynote speakers) with over 170,000 researchers who had been the last authors on computational biology papers between 1993 and 2019.
The proportion of female fellows and keynote speakers was similar to that of the field overall.
However, names of East Asian origin have been persistently underrepresented among fellows and keynote speakers.
Moreover, fellows and keynote speakers with an affiliation in the United States were overrepresented by a factor of 2.0;
almost three quarters of this excess was accounted for by a deficit of 89 fellows and keynote authors from China, India, France and Italy.
Within the US, we found an excess of White fellows and keynote speakers and a depletion of Asian fellows and keynote speakers.

<!-- Short abstract
Delivering a keynote talk at a conference or being named as a fellow by a scientific society indicates that a scientist is held in high regard by their colleagues.
To explore if the distribution of such indicators of esteem in the field of bioinformatics reflects the composition of this field, 
we compared the gender, name-origin, and country of affiliation of 412 researchers who had been recognized by the International Society for Computational Biology (honorees) with 34,050 researchers who had been the corresponding authors on papers in three leading bioinformatics journals between 1993 and 2019.
The proportion of female honorees was similar to that of the field.
However, names of East Asian origin have been persistently underrepresented among honorees.
Moreover, honorees with an affiliation in the US were overrepresented by a factor of 2.0.
Within the US, we found an excess of White honorees and a depletion of Asian honorees.
-->


## Introduction

Scientists' roles in society include identifying important topics of study, undertaking an investigation of those topics, and disseminating their findings broadly.
The scientific enterprise is largely self-governing: scientists act as peer reviewers on papers and grants, comprise hiring committees in academia, make tenure decisions, and select which applicants will be admitted to doctoral programs.
A lack of diversity in science could lead to pernicious biases that hamper the extent to which scientific findings are relevant to minoritized communities.
Furthermore, even though minoritized groups innovate at higher rates, their novel contributions are discounted [@doi:10.1073/pnas.1915378117].
One first step to address this systemic issue is to directly examine peer recognition in different scientific fields.

Gender bias among conference speakers has been recognized as an area that can be improved with targeted interventions [@doi:10.1371/journal.pcbi.1003903; @doi:10.1126/science.caredit.aaw9742; @doi:10.1038/ni.3707; @arxiv:1502.06326].
Having more female organizers on conference committees is associated with having more female speakers [@doi:10.1128/mBio.00846-13].
At medical conferences in the US and Canada, the proportion of female speakers is increasing at a modest rate [@doi:10.1001/jamanetworkopen.2019.2103].
Gender bias appears to also influence funding decisions: an examination of scoring of proposals in Canada found that reviewers asked to assess the science produced a smaller gender gap in scoring than reviewers asked to assess the applicant [@doi:10/djc5].

Challenges extend beyond gender: an analysis of awards at the NIH found that proposals by Asian, black or African-American applicants were less likely to be funded than those by White applicants [@doi:10.1126/science.1196783].
There are also potential interaction effects between gender and race or ethnicity that may particularly affect women of color's efforts to gain NIH funding [@doi:10.1097/ACM.0000000000001278].
Another recent analysis found that minority scientists tend to apply for awards on topics with lower success rates [@doi:10.1126/sciadv.aaw7238].
This finding might be the result of minority scientists selecting topics in more poorly funded areas.
Alternatively, reviewing scientists may not recognize the scientific importance of these topics, which may be of particular interest to minority scientists.

We sought to understand the extent to which honors and high-profile speaking invitations were distributed equitably among gender and name origin groups by an international society and its associated meetings.
As computational biologists, we focused on the [International Society for Computational Biology](https://www.iscb.org/) (ISCB), its honorary Fellows as well as its affiliated international meetings that aim to have a global reach: [Intelligent Systems for Molecular Biology](https://en.wikipedia.org/wiki/Intelligent_Systems_for_Molecular_Biology) (ISMB) and [Research in Computational Molecular Biology](https://en.wikipedia.org/wiki/Research_in_Computational_Molecular_Biology) (RECOMB).

We used multiple methods to predict the gender and name origins of honorees.
Existing methods were relatively US-centric because most of the data was derived in whole or in part from the US Census.
We scraped more than 700,000 entries from English-language Wikipedia that contained nationality information to complement these existing methods and built multiple machine learning classifiers to predict name origin.
We also examined the last authors for more than 170,000 computational biology publications to establish a field-specific baseline using the same metrics.
The results were consistent across all approaches: we found a dearth of non-White speakers and honorees.
The lack of Asian scientists among keynote speakers and Fellows was particularly pronounced when compared against the field-specific background.


## Results

We curated a dataset of ISCB honorees that included 412 honorees who were keynote speakers at international ISCB-associated conferences (ISMB and RECOMB) as well as ISCB Fellows.
The ISCB Fellows set contained the complete set of Fellows named (2009--2019).
Keynote speakers were available for ISMB for all years from 1993--2019.
Keynote speakers for RECOMB were available for all years from 1997--2019.
We included individuals who were honored multiple times as separate entries.
For example, Christine Orengo was a keynote speaker at RECOMB 2004 and became an ISCB Fellow in 2016, and thus was counted twice in this list.

We sought to compare this dataset with a background distribution of potential speakers, which we considered to be last authors of bioinformatics and computational biology manuscripts.
We scraped PubMed for manuscripts written in English from 1993--2019 with the MeSH term ["computational biology"](https://pubmed.ncbi.nlm.nih.gov/?term=%22Computational+Biology%22%5BMeSH+Terms%5D+).
We downloaded the metadata of manuscripts published in these journals from PubMed, which provided 176,110 articles for evaluation.
For each article, we extracted its last author's fore name and last name for analysis.

### Technical considerations

Selecting an optimal study design is the important first step to acquire an accurate and thorough answer to any research question.
This study is not an exception: we were faced with multiple decisions at many analysis steps.
We are grateful to all the reviewers whose input made the study more rigorous after each revision.
As we discussed and revised the analysis, we committed to stay transparent and documented the reasoning and potential caveats of each decision.
Details of these discussions can be found under the [Issues page](https://github.com/greenelab/iscb-diversity-manuscript/issues?q=is%3Aissue+is%3Aclosed) on the study GitHub repository.
We highlighted here four important decisions we made and how each one may affect the result (Fig. {@fig:versions}).

![
Study progression.
In [version 1](https://greenelab.github.io/iscb-diversity-manuscript/v/711edb01f5335a4c1bb8e1e0575618586a7abf9b/) of this work, instead of last authors, we examined corresponding authors of articles in three well-recognized computational biology journals, 
and the results were consistent with our findings in [version 2](https://greenelab.github.io/iscb-diversity-manuscript/v/6bda862974e77beb98697f39c18027683af158cb/), where we expanded the article dataset to include all PubMed articles with the MeSH terms "computational biology". 
Finally, in this [latest version](https://greenelab.github.io/iscb-diversity-manuscript/), we used citation count to weight articles to account for the impact variation of research contributions.
](https://raw.githubusercontent.com/greenelab/iscb-diversity/a61d67806a5212488b4c574078f80467cbd8cf5d/figs/versions.png){#fig:versions width="90%"}

Our findings across different versions of the study were generally consistent.
We first expanded the article set from 29,755 journal-specific to 176,110 all computational biology PubMed articles, making the sample size nearly six times larger than previously.
We also extracted names of last authors instead of other potential selections to better capture the honoree population.
Our assumption was that, among the authors of a specific paper, the last author (often research advisors) would be most likely to be invited for a keynote or honored as a fellow.
Also, the availability of information on corresponding authors was limited, and extracting last author became a more consistent approach.
However, we acknowledged that we might have miss corresponding authors in the first position who can be considered for keynote speakers or fellows [@doi:10.1177/0306312716650046].

In version 3, instead of considering all articles equally as in the earlier versions, we used citation count to weight articles to control for the differential impact of research contributions.
However, we note a potential bias with the use of citation counts: female scientists of different disciplines are cited less often than their male counterparts [@arxiv:2001.01002; @doi:10.1002/ece3.4993; @doi:10.1038/s41550-017-0141].
To the extent that there may be a bias in citation practices, we should expect that the bias for honorees would align with one for citations, meaning that our analysis would understate the severity of disparities.
Furthermore, the act of being honored, particularly with a keynote at an international meeting, could lead to work being more recognized and cited, which would reverse the arrow of causality.
This would also lead to our analyses understating the difference.
Previously in version 1--2, without citation counts, we found even more pronounced differences between the two groups of authors and honorees.

Finally, in all versions of the analysis, rather than applying a hard assignment for each prediction, we analyzed the raw prediction probability values to capture the uncertainty of the prediction model, improving the accuracy of the aggregate statistics.
Although our final estimates of the effect size may be conservative (see more in Discussion), the general conclusion remained robust throughout different sets of study design parameters.
If other researchers aim to answer similar questions of their field, our recommendation is to take caution when making these decisions and account for field-specific factors such as authorship practice.

### Similar gender proportion between ISCB's honorees and the field

We observed a gradual increase of the proportion of predicted female authors, arriving at an average of approximately 27% in 2017-2019 (Fig. {@fig:gender_breakdown}, left).
In recent years, ISCB Fellows and keynote speakers appear to have similar gender proportions compared to the population of authors published in computational biology and bioinformatics journals (averaged around 30% in the last three years, Fig. {@fig:gender_breakdown}, right).
Examining each honor category, we observed an increasing trend of honorees who were women, especially in the group of ISCB Fellows (see [notebook](https://greenelab.github.io/iscb-diversity/10.visualize-gender.html#sup_fig_s1)), which markedly increased after 2015.
Through 2019, there were a number of years when meetings or ISCB Fellow classes have a high probability of recognizing only male honorees and none that appeared to have exclusively female honorees.
We sought to examine whether or not there was a difference in the proportion of female names between authors and honorees. 
A multiple linear regression of this proportion for the groups and year did not reveal a significant difference (_p_ = 0.237).
Interaction terms did not predict the group of scientists over and above the main effect of gender probability and year.

![Estimated composition of gender prediction over the years of
  all Pubmed computational biology and bioinformatics journal authors (left),
  and all ISCB Fellows and keynote speakers (right).
  Male proportion (yellow) was computed as the average of the probability of being male of last authors (weight accordingly) or ISCB honorees each year.
  Female proportion (blue) was the complement of the male proportion.
  ISCB Fellows and keynote speakers appear have similar gender proportions compared to that of PubMed authors.
](https://raw.githubusercontent.com/greenelab/iscb-diversity/ac43b098a2a994cbe3788bf7a19f541fc533b3de/figs/gender_breakdown.png){#fig:gender_breakdown width="70%"}


### Honorees with Celtic/English names are overrepresented while honorees with East Asian names are underrepresented

We applied our Wiki2019-LSTM model to both our computational biology honorees dataset and our dataset of last authors.
We found that the proportion of authors with Celtic/English names had decreased (Fig. {@fig:region_breakdown}A, left).
Among keynote speakers and fellows, we found that the majority of honorees are predicted to have Celtic/English or European names (Fig. {@fig:region_breakdown}A, right).
When we directly compared honoree composition with PubMed, we observed discrepancies between the two groups (Fig. {@fig:region_breakdown}B).
Compared to other names, a Celtic/English name has significantly higher odds of being selected as an honoree (OR~Celtic/English~ = 1.21, $\beta$~Celtic/English~ = 0.19, _p_ = 6.7×10^-4^).
Meanwhile, an East Asian name has significantly lower odds of being selected as an honoree compared to other names (OR~East\ Asian~ = 0.19, $\beta$~East\ Asian~ = -1.67, _p_ = 5.9×10^-9^).
The two groups of scientists did not have a significant association with names predicted to be European and in Other categories (_p_ = 0.37 and _p_ = 0.44, respectively).
Interaction terms did not predict the group of scientists over and above the main effect of name origin probability and year.

![(A) Estimated composition of name origin prediction over the years of
  all Pubmed computational biology and bioinformatics journal authors (left),
  and all ISCB Fellows and keynote speakers (right).
  (B) For each region, the mean predicted probability of Pubmed articles is shown as teal generalized additive model curve, and the mean probability and 95% confidence interval of the ISCB honoree predictions are shown as dark circles and vertical lines.
  Compared to the name collection of Pubmed authors, honorees with Celtic/English names are overrepresented while honorees with East Asian names are underrepresented.
  No statistically significant difference was observed between the proportion of honorees and authors with European names or in other categories (see Table @tbl:example_names).

](https://raw.githubusercontent.com/greenelab/iscb-diversity/ac43b098a2a994cbe3788bf7a19f541fc533b3de/figs/region_breakdown.png){#fig:region_breakdown}

When applying the Wiki2019-LSTM model to the name origins of only US-affiliated scientists, we found a similar underrepresentation of honorees with East Asian names (OR~East\ Asian~ = 0.18, $\beta$~East\ Asian~ = -1.71, _p_ = 6.5×10^-4^).
We observed no statistically significant difference between the proportion of honorees and authors with Celtic/English names, European names, or names in Other categories (_p_ = 0.61, _p_ = 0.12, and _p_ = 0.72, respectively).
We note that the US was not included in the training of the Wiki2019-LSTM model (see Methods).
Please see [analysis notebook](https://greenelab.github.io/iscb-diversity/14.us-name-origin.html) for more details.


### Overrepresentation of US-affiliated honorees

We analyzed the countries of affiliation between last authors and ISCB honorees.
For each country, we report a value of log~2~ enrichment (LOE) and its 95% confidence intervals.
The full table with all countries and their corresponding enrichment can be browsed interactively in the corresponding [analysis notebook](https://greenelab.github.io/iscb-diversity/12.analyze-affiliation.html#enrichment_tab).
A positive value of LOE indicates a higher proportion of honorees affiliated with that country compared to authors.
A LOE value of 1 represents a one-fold enrichment (i.e., observed number of honorees is twice as much as expected).
In the 20 countries with the most publications, we found an overrepresentation of honorees affiliated with institutions and companies in the US (119 speakers more than expected, LOE = 1.0, 95% CI (0.8, 1.2)) and Israel (15 speakers more than expected, LOE = 2.7, 95% CI (1.9, 3.4)), and an underrepresentation of honorees affiliated with those in China, France, Italy, India, South Korea, and Brazil (Fig. @fig:country-enrichment).

![Each country's log~2~ enrichment (LOE) and its 95% confidence interval (left),
and the absolute difference between observed (triangle) and expected (circle) number of honorees (right).
Positive value of LOE indicates a higher proportion of honorees affiliated with that country compared to authors.
Countries are ordered based on the proportion of authors in the field.
The overrepresentation of honorees affiliated with institutions and companies in the US and Israel contrasts the underrepresentation of honorees affiliated with those in China, France, Italy, India, South Korea, and Brazil.

](https://raw.githubusercontent.com/greenelab/iscb-diversity/ac43b098a2a994cbe3788bf7a19f541fc533b3de/figs/enrichment-plot.png){#fig:country-enrichment width="80%"}


### Improvements to Honoree Diversity Subsequent to Our Primary Analysis

While our study was primarily designed to assess the diversity of honorees, the findings raise important questions about what can be done to address the disparities.
By publishing our original findings on the biorxiv preprint server, we could begin to answer these questions by examining changes subsequent to our initial report.
We released [version 1.0](https://github.com/greenelab/iscb-diversity-manuscript/releases/tag/v1.0) of our manuscript on 2020-01-30.
Early indications suggest ISCB is now increasing the diversity of honorees.
In 2020, among 12 ISCB Fellows and 5 ISMB keynote speakers, the mean predicted probability of each honoree having an East Asian name is 33%, higher than any estimate in previous years (see [notebook](https://greenelab.github.io/iscb-diversity/15.analyze-2020.html)).
The set of honorees also includes the first ISCB Fellow from China.
Compared to past years, the 2020 honorees appear to better reflect the diversity of scientists in the computational biology field.
These new results suggest: 1) deserving honorees who were members of under-recognized groups existed but had not been recognized until now, and 2) simply examining honoree distribution's alignment with the field can trigger changes that address issues of unequal representation.
However, we note that this analysis dealt only with more senior scientists (the last authors on scientific manuscripts) in the context of honors and that many years of changed honoree distributions will be required for the set of honored scientists to better reflect the field's senior author contributions.


## Discussion

A major challenge that we faced in carrying out this work was to narrow down geographic origins for some groups of names.
Some name origin groups, such as Hispanic names, are geographically disparate.
We were unable to construct a classifier that could distinguish between names from Iberian countries (Spain and Portugal) from those in Latin America in the group of Hispanic names.
Discrepancies in representation between these groups are thus undetectable by our classifier.
Honoree counts of those with Hispanic names are influenced from Spain as well as Latin America.
In such cases, our analyses may substantially understate the extent to which minoritized scientists are underrepresented among honorees and authors.

Biases in authorship practices may also result in our underestimation of the composition of minoritized scientists within the field.
We estimated the composition of the field using last author status, but in neuroscience [@doi:10.1101/275362] and other disciplines [@doi:10.1371/journal.pbio.2004956] women are underrepresented among such authors.
Such an effect would cause us to underestimate the number of women in the field.
Similarly, other studies have showed that underrepresented groups are less likely to be last authors [@doi:10.1257/pandp.20181111], and Hispanic and Black scientists were underrepresented in academic publishing in general [@doi:10.1007/s11192-012-0893-4].

We acknowledged that our supervised learning approaches are neither error free nor bias free.
By integrating different methods and preserving uncertainty by analyzing prediction probabilities rather than applying a hard assignment for each prediction, we hope to alleviate these biases and discover insightful findings that correctly reflect the current representation diversity at conferences.
Because of the bias in citation practices, potentially overestimated number of US-affiliated authors, ambiguity in geographic origins of Hispanic names, our estimation would render a conservative one.
In other words, it is likely that the actual underrepresentation of East Asian honorees and overrepresentation of US-affiliated honorees is more severe, and there can be an underrepresentation of women and other minoritized scientists if the estimates of the proportions of these scientists in the field were more accurate.

Focusing on an international society and meetings, we measured honor and authorship rates worldwide.
In this setting, we observe disparities by name groups.
Because invitation and honor patterns could be driven by biases associated with name groups, geography, or other factors, we cross-referenced name group predictions with author affiliations to disentangle the relationship between geographic regions, name groups and invitation probabilities.
We found that disparities persisted even within the group of honorees with a US affiliation.

An important questions to ask when measuring representation is what the right level of representation is.
We suggest that considering equity may be more appropriate than strictly diversity.
For example, we found similar representation of women between authors and honorees, which suggests honoree diversity is similar to that of authors.
However, if fewer women are in the field because of systemic factors that inhibit their participation, we would not have reached equity.
In addition to holding fewer corresponding authorship positions, on average, female scientists of different disciplines are cited less often [@arxiv:2001.01002, @doi:10.1002/ece3.4993, @doi:10.1038/s41550-017-0141, @pmid:24350369], invited by journals to submit papers less often [@doi:10.1371/journal.pbio.2004956], suggested as reviewers less often [@doi:10.1038/541455a], and receive significantly worse review scores [@doi:10.1002/ece3.4993].
Meanwhile, a review of women's underrepresentation in math-intensive fields argued that today's underrepresentation is not explained by historic forms of discrimination but factors surrounding fertility decisions and lifestyle choices, whether freely made or constrained by biology and society [@doi:10.1073/pnas.1014871108].
A recent analysis of gender inequality across different disciplines showed that, although both gender groups have equivalent annual productivity, women scientists have higher dropout rates throughout their scientific careers [@doi:10.1073/pnas.1914221117].
Therefore, although we found that ISCB's honorees and keynote speakers appear to have similar gender proportion to the field as a whole, the gender proportions have not reached parity due to these systemic inequalities that obstruct women's progress in science.

Additionally, we found that honorees include significantly fewer people of color than the field as a whole, and Asian scientists are dramatically underrepresented among honorees.
Societies' honoree selection process failing to reflect the diversity of the field can play a part in why minoritized scientists' innovations are discounted [@doi:10.1073/pnas.1915378117].
Although we estimate the fraction of non-White and non-Asian authors to be relatively similar to the estimated honoree rate, we note that both are represented at levels substantially lower than in the US population.

Societies, both through their honorees and the individuals who deliver keynotes at their meetings, can play a positive role in improving the presence of female STEM role models, which can boost young students' interests in STEM [@doi:10.1073/pnas.1014871108] and, for example, lead to higher persistence for undergraduate women in geoscience [@doi:10.1130/GES01659.1].
Efforts are underway to create Wikipedia entries for more female [@doi:10.1038/d41586-018-05947-8] and black, Asian, and minority scientists [@doi:10.1038/d41586-019-00812-8], which can help early-career scientists identify role models.
Societies can contribute toward equity if they design policies to honor scientists in ways that counter these biases such as ensuring diversity in the selection committees.

The central role that scientists play in evaluating each other and each other's findings makes equity critical.
Even many nominally objective methods of assessing excellence (e.g., h-index, grant funding obtained, number of high-impact peer-reviewed publications, and total number of peer-reviewed publications) are subject to the bias of peers during review.
These could be affected by explicit biases, implicit biases, or pernicious biases in which a reviewer might consider a path of inquiry, as opposed to an individual, to be more or less meritorious based on the reviewer's own background [@doi:10.1126/sciadv.aaw7238, @doi:10.1101/400515].
Our efforts to measure the diversity of honorees in an international society suggests that, while a focus on gender parity may be improving some aspects of diversity among honorees, contributions from scientists of color are underrecognized.


## Materials and Methods

### Honoree Curation

From [ISCB's webpage listing **ISCB Distinguished Fellows**](http://web.archive.org/web/20200116150052/https://www.iscb.org/iscb-fellows), we found recipients listed by their full names for the years 2009--2019.
We gleaned the full name of the Fellow as well as the year in which they received the honor.
We identified major ISCB-associated conferences as those designated flagship (ISMB) or those that had been held on many continents (RECOMB).
To identify **ISMB Keynote Speakers**, we examined the webpage for each ISMB meeting.
The invited speakers at ISMB before 2001 were listed in the Preface pages of each year's proceedings, which were archived in the [ISMB collection](https://aaai.org/Library/ISMB/ismb-library.php) of the AAAI digital library.
We found full names of all keynote speakers for the years 1993--2019.

For the RECOMB meeting, we found conference webpages with keynote speakers for 1999, 2000, 2001, 2004, 2007, 2008, and 2010--2019.
We were able to fill in the missing years using information from the RECOMB 2016 proceedings, which summarizes the first 20 years of the RECOMB conference [@doi:10.1007/978-3-319-31957-5].
This volume has two tables of keynote speakers from 1997--2006 (Table 14, page XXVII) and 2007--2016 (Table 4, page 8).
Using these tables to verify the conference speaker lists, we arrived at two special instances of inclusion/exclusion.
Although Jun Wang was not included in these tables, we were able to confirm that he was a keynote speaker in 2011 with the RECOMB 2011 proceedings [@doi:10.1007/978-3-642-20036-6], and thus we included this speaker in the dataset.
Marian Walhout was invited as a keynote speaker but had to [cancel](http://recomb2015.mimuw.edu.pl/node/18.html) the talk due to other obligations.
Because her name was neither mentioned in the 2015 proceedings [@doi:10.1007/978-3-319-16706-0] nor in the above-mentioned tables, we excluded this speaker from our dataset.

#### Name processing

When extracting honoree names, we began with the full name as provided on the site.
Because our prediction methods required separated first and last names, we chose the first non-initial name as the first name and the final name as the last name.
We did not consider a hyphen to be a name separator:
for hyphenated names, all components were included.
For metadata from PubMed and PMC where first (fore) and last names are coded separately, we applied the same cleaning steps.
We created [functions to simplify names](https://git.dhimmel.com/pubmedpy/names.html) in the pubmedpy Python package to support standardized fore and last name processing.


### Last author extraction

We assumed that, in the list of authors for a specific paper, last authors (often research advisors) would be most likely to be invited for keynotes or to be honored as fellows.
Therefore, we utilized [PubMed](https://pubmed.ncbi.nlm.nih.gov/) to retrieve last author names to assess the composition of the field.
PubMed is a search engine resource provided by the US National Library of Medicine and index scholarly articles.
PubMed contains a record for every article published in journals it indexes (30 million records total circa 2020), within which we were able to extract author first and last names and their order using the E-Utilities APIs.
To automate and generalize these tasks, we created the [pubmedpy](https://github.com/dhimmel/pubmedpy) Python package.

From PubMed, we compiled a catalog of 176,773 journal articles that were published from 1993 through 2019 that were written in English and tagged with the MeSH term ["computational biology"](https://pubmed.ncbi.nlm.nih.gov/?term=%22Computational+Biology%22%5BMeSH+Terms%5D+), which is [equivalent](https://www.ncbi.nlm.nih.gov/mesh/68019295) to "bioinformatics" and includes categories such as genomics and systems biology (via PubMed's term explosion to include subterms).
Excluding 663 articles with no author information and years with less than 200 articles/year, we analyzed 176,110 articles from 1998--2019.
We extracted the number of times an article has been cited by PubMed Central articles from the `PmcRefCount` of the PubMed DocSum XML records.
Under the rationale that authors of more impactful articles have higher chance of being recognized, we used the PubMed Central citation count as a relative estimate of an article's impact to weigh each author's prediction probability (please see the Statistical Analysis subsection).

### Countries of Affiliations

Publications often provide affiliation lists for authors, which generally associate authors with research organizations and their corresponding physical addresses.
We implemented affiliation extraction in the pubmedpy Python package for both PubMed and PMC XML records.
These methods extract a sequence of textual affiliations for each author.

We relied on two Python utilities to extract countries from text: [geotext](https://github.com/elyase/geotext) and [`geopy.geocoders.Nominatim`](https://geopy.readthedocs.io/en/stable/#nominatim).
The first, geotext, used regular expressions to find mentions of places from the [GeoNames database](http://www.geonames.org/).
To avoid mislabeling, we only mapped the affiliation to a country if geotext identified 2 or more mentions of that country.
For example, in the affiliation string "Laboratory of Computational and Quantitative Biology, Paris, France", geotext detected 2 mentions of places in France: Paris, France.
In this case, we assign France to this affiliation.

This country extraction method accommodates multiple countries.
Although ideally each affiliation record would refer to one and only one research organization, sometimes journals deposit multiple affiliations in a single structured affiliation.
In these cases, we assigned multiple countries to the article.
For more details on this approach, please consult the accompanying [notebook](https://github.com/greenelab/iscb-diversity/blob/ac43b098a2a994cbe3788bf7a19f541fc533b3de/07.affiliations-to-countries.ipynb) and [label dataset](https://github.com/greenelab/iscb-diversity/blob/ac43b098a2a994cbe3788bf7a19f541fc533b3de/data/affiliations/geocode.jsonl).

When geotext did not return results, we use the geopy approach, which returns a single country for an affiliation when successful.
Its `geocoders.Nominatim` function converts names / addresses to geographic coordinates using the OpenStreetMap's [Nomatim](https://nominatim.org/) service.
With this method, we split a textual affiliation by punctuation into a list of strings and iterate backward through this list until we found a Nomatim search result.
For the above affiliation, the search order would be "France", "Paris", and "Laboratory of Computational and Quantitative Biology".
Since Nomatim would return a match for the first term "France" (matched to France), the search would stop before getting to "Paris", and "France" would be assigned to this affiliation.

Our ability to assign countries to authors was largely driven by the availability of affiliations.
The country-assignment-rate for last authors from PubMed records was approximately 47%. 
This reflects the varying availability of affiliation metadata by journal.

For ISCB honorees, during the curation process, if an honoree was listed with their affiliation at the time, we recorded this affiliation for analysis.
For ISCB Fellows, we used the affiliation listed on the ISCB page.
Because we could not find affiliations for the 1997 and 1998 RECOMB keynote speakers' listed for these years, they were left blank.
If an author or speaker had more than one affiliation, each was inversely weighted by the number of affiliations that individual had.

### Estimation of Gender

We predicted the gender of honorees and authors using the <https://genderize.io> API, which was trained on over 100 million name-gender pairings collected from the web and is one of the three widely-used gender inference services [@doi:10.7717/peerj-cs.156].
We used author and honoree first names to retrieve predictions from genderize.io.
The predictions represent the probability of an honoree or author being male or female.
We used the estimated probabilities and did not convert to a hard group assignment.
For example, a query to <https://genderize.io> on January 26, 2020 for "Casey" returns a probability of male of 0.74 and a probability of female of 0.26, which we would add for an author with this first name.
Because of technical limitations, our analysis only considered two binary gender categories, and we used "male" and "female" to refer to the gender of the scientists.
However, we recognized the limitation of not accounting for non-binary gender categories and only considered predictions in aggregate and not as individual values for specific scientists.

Of 412 ISCB honorees, genderize.io fails to provide gender predictions for one name.
Of 176,110 last authors, 1,014 were missing a fore name in the raw paper metadata and 11,498 had fore names consisting of only initials.
Specifically, the metadata for most papers before 2002 (2,566 out of 2,601 papers) only have initials for first and/or middle author names.
Without gender predictions for these names, we consider only articles from 2002 on when comparing gender compositions between two groups.
Of the remaining authors, genderize.io failed to predict gender for 10,003 of these fore names.
We note that approximately 42% of these NA predictions are hyphenated names, which is likely because they are more unique and thus are more difficult to find predictions for.
<!-- 82% of these names were predicted to be of Asian origin by last name (see the race/ethnicity prediction model below). -->
This bias of NA predictions toward non-English names has been previously observed [@doi:10.32614/RJ-2016-002] and may have a minor influence on the final estimate of gender compositions.

### Estimation of Name Origin Groups

We developed a model to predict geographical origins of names.
The existing Python package ethnicolr [@arxiv:1805.02109] produces reasonable predictions, but its international representation in the data curated from Wikipedia in 2009 [@doi:10.1145/1557019.1557032] is still limited.
For instance, 76% of the names in ethnicolr's Wikipedia dataset are European in origin.

To address these limitations in ethnicolr, we built a similar classifier, a Long Short-term Memory (LSTM) neural network, to infer the region of origin from patterns in the sequences of letters in full names.
We applied this model on an updated, approximately 4.5 times larger training dataset called Wiki2019 (described below).
We tested multiple character sequence lengths and, based on this comparison, selected tri-characters for the primary results described in this work.
We trained our prediction model on 80% of the Wiki2019 dataset and evaluated its performance using the remaining 20%.
This model, which we term Wiki2019-LSTM, is available in the online file [`LSTM.h5`](https://github.com/greenelab/wiki-nationality-estimate/blob/7554e8a124760582e22ff9b051433729b49c22a9/models/LSTM.h5).

To generate a training dataset for name origin prediction that reflects a modern naming landscape, we scraped the English Wikipedia's category of [Living People](https://en.wikipedia.org/wiki/Category:Living_people).
This category, which contained approximately 930,000 pages at the time of processing in November 2019, is regularly curated and allowed us to avoid pages related to non-persons.
For each Wikipedia page, we used two strategies to find a full birth name and location context for that person.
First, we looked for nationality mention in the first sentence in the body of the text.
In most English-language biographical Wikipedia pages, the first sentence usually begins with, for example, "John Edward Smith (born 1 January 1970) is an American novelist known for ..."
This structure comes from editor [guidance on biography articles](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Biography#Context) and is designed to capture:

> ... the country of which the person is a citizen, national or permanent resident, or if the person is notable mainly for past events, the country where the person was a citizen, national or permanent resident when the person became notable.

Second, if this information is not available in the first sentence of the main text, we used information from the personal details sidebar; the information in this sidebar varied widely but often contained a full name and a place of birth.

We used regular expressions to parse out the person's name from this structure and checked that the expression after "is a" matched a list of nationalities.
We were able to define a name and nationality for 708,493 people by using the union of these strategies.
This process produced country labels that were more fine-grained than the broader patterns that we sought to examine among honorees and authors.
We initially grouped names by continent, but later decided to model our categorization after the hierarchical taxonomy used by [NamePrism](http://www.name-prism.com/about) [@doi:10.1145/3132847.3133008].
The NamePrism taxonomy was derived from name-country pairs by producing an embedding of names by Twitter contact patterns and then grouping countries using the similarity of names from those countries.
The countries associated with each grouping are shown in Fig {@fig:nameprism_countries}.
NamePrism excluded the US, Canada and Australia because these countries have been populated by a mix of immigrant groups [@doi:10.1145/3132847.3133008].

![NamePrism groups countries by name similarity. We used this grouping but renamed the groups to focus on the linguistic patterns based on name etymology identified by NamePrism.
](https://raw.githubusercontent.com/greenelab/iscb-diversity/ac43b098a2a994cbe3788bf7a19f541fc533b3de/figs/2020-01-31_groupings.png){#fig:nameprism_countries width="90%"}

In an earlier version of this manuscript, we also used category names derived from NamePrism, but a reader [pointed out](https://github.com/greenelab/iscb-diversity-manuscript/issues/27) the titles of the groupings were problematic;
therefore, in this version, we renamed these groupings to reflect that the NamePrism approach primarily identifies groups based on linguistic patterns from name etymology rather than religious or racial similarities.
We note that our mapping from nationality to name origins was not without error.
For example, a scientist of Israeli nationality may not bear a Hebrew name.
These mismatches were assessed via the heatmap of the model performance (Fig. {@fig:wiki2019_lstm}C) and complemented by the affiliation analysis below.
An alternative approach is to assign arbitrary names to these groups such as via letter coding (e.g., A, B, C, etc.), but we did not choose this strategy because ten arbitrary letters for ten groups can greatly reduce the paper's readibility.


### Predicting Name Origin Groups with LSTM Neural Networks and Wikipedia

Table @tbl:example_names shows the size of the training set for each of the name origin groups as well as a few examples of PubMed author names that had at least 90% prediction probability in that group.
We refer to this dataset as Wiki2019 (available online in [`annotated_names.tsv`](https://github.com/greenelab/wiki-nationality-estimate/blob/master/data/annotated_names.tsv)).

| Group | Training Size | Example Names |
| ------ | ------ | -------------------- |
| Celtic/English names | 154,890 | Adam O Hebb, Oliver G Pybus, David W Ritchie, James WJ Anderson, James W MacDonald, Robert Clarke |
| European names | 78,157 | Tracey M Filzen, Jos H Beijnen, Caroline Louis-Jeune, Christian Lorenzi, Boris Vassilev, Verena Heinrich |
| Hispanic names | 66,931 | Ramón Latorre, Antonio J Jimeno-Yepes, Felipe A Simão, Paulo S L de Oliveira, Juan Carlos Rodríguez-Manzaneque, Natalia Acevedo-Luna |
| East Asian names | 54,588 | Heejoon Chae, Wenchao Jiang, Haizhou Liu, Miho Uchida, Wenxuan Zhang, Jiali Feng |
| Arabic/Turkic/Persian names | 31,418 | Hamidreza Chitsaz, Farzad Sangi, Habib Motieghader, Berke Ç Toptas, Ali Aliyari, Bülent Arman Aksoy |
| Nordic names | 28,978 | Cecilia M Lindgren, Ellen Larsen, Jesper R Gådin, Janne H Korhonen, Johan Åqvist, Jens Nilsson |
| South Asian names | 20,025 | Amitabh Chak, Matthew G Seetin, Matrika Gupta, Sumudu P Leelananda, VS Kumar Kolli, Swanand Gore |
| African names | 17,826 | Timothy Kinyanjui, Jammbe Z Musoro, Nyaradzo M Mgodi, Magambo Phillip Kimuda, Probhonjon Baruah, Adaoha E C Ihekwaba |
| Hebrew names | 4,549 | Alexander J Sadovsky, Boris Shraiman, Gil Goldshlager, Eytan Adar, Aviva Peleg, Nir Esterman |
| Greek names | 4,138 | Gianni Panagiotou, Themis Lazaridis, Eleni Mijalis, Nikolaos Tsiantis, Konstantinos A Kyritsis, Dimitris E Messinis |

Table: **Predicting name-origin groups of names trained on Wikipedia's living people.**
The table lists the 10 groups and the number of living people for each region that the LSTM was trained on.
Example names shows actual author names that received a high prediction for each region.
Full information about which countries comprised each region can be found in the online dataset [`country_to_region.tsv`](https://github.com/greenelab/iscb-diversity/blob/ac43b098a2a994cbe3788bf7a19f541fc533b3de/data/countries/2020-01-31_groupings.tsv).
{#tbl:example_names}

We next aimed to predict the name origin groups of honorees and authors.
We constructed a training dataset with more than 700,000 name-nationality pairs by parsing the English-language Wikipedia.
We trained a LSTM neural network on n-grams to predict name groups.
We found similar performance across 1, 2, and 3-grams;
however, because the classifier required fewer epochs to train with 3-grams, we used this length in the model that we term Wiki2019-LSTM.
Our Wiki2019-LSTM returns, for each given name, a probability of that name originating from each of the specified 10 groups.
We observed a multiclass area under the receiver operating characteristic curve (AUC) score of 95.9% for the classifier, indicating that the classifier can recapitulate name origins with high sensitivity and specificity.
For each individual group, the high AUC (above 92%, Fig. {@fig:wiki2019_lstm}A) suggests that our classifier was sufficient for use in a broad-scale examination of disparities.
We also observed that the model was well calibrated (Fig. {@fig:wiki2019_lstm}B).
We also examined potential systematic errors between pairs of name origin groupings with a confusion heatmap and did not find off-diagonal enrichment for any pairing (Fig. {@fig:wiki2019_lstm}C).

![The Wiki2019-LSTM model performs well on the testing dataset.
The area under the ROC curve is above 92% for each category, showing strong performance across origin categories (A).
A calibration curve, computed with the caret R package, shows consistency between the predicted probabilities (midpoints of each fixed-width bin) and the observed fraction of names in each bin (B).
Heatmap showing whether names from a given group (x-axis) received higher (purple) or lower (green) predictions for each group (y-axis) than would be expected by group prevalence alone (C).
The values represent log~2~ fold change between the average predicted probability and the prevalence of the corresponding predicted group in the testing dataset (null).
Scaling by group prevalence accounts for the imbalance of groups in the testing dataset.
In all cases, the classifier predicts the true groups above the expected null probability (matrix diagonals are all purple).
For off-diagonal cells, darker green indicates a lower mean prediction compared to the null.
For example, the classifier does not often mistake East Asian names as Greek, but is more prone to mistaking South Asian names as Celtic/English.
](https://raw.githubusercontent.com/greenelab/iscb-diversity/ac43b098a2a994cbe3788bf7a19f541fc533b3de/figs/fig_3.png){#fig:wiki2019_lstm width=100%}

Applying Wiki2019-LSTM on the author and honoree datasets, we obtained name origin estimates for all honorees' and authors' name, except the 12,512 that did not have fore names (see breakdown in the Estimation of Gender section above).
Once again, because the large majority of author fore names prior to 2002 were recorded with initials only, predictions were not possible, and we excluded 1998--2001 when comparing name origin compositions between two groups.

### Affiliation Analysis

For each country, we computed the expected number of honorees by multiplying the proportion of authors whose affiliations were in that country with the total number of honorees.
We then performed an enrichment analysis to examine the difference in country affiliation proportions between ISCB honorees and field-specific last authors.
We calculated each country's enrichment by dividing the observed proportion of honorees by the expected proportion of honorees.
The 95% confidence interval of the log~2~ enrichment was estimated using the Poisson model method [@isbn:978-0849394447, page 17-18].


### Statistical Analysis

We estimated the levels of representation by performing the following logistic regression of the group of scientists on each name's prediction probability while controlling for year:
$$g = \beta_0 + \beta_1 prob' + \beta_2 y + \epsilon.$$
The variable $prob'$ is the adjusted prediction probablity of a demographic variable (gender and name origin) for names of scientists in group $g$ (honoree or author) during year $y$.
$\epsilon \sim N(0, \sigma^2)$ accounts for random variation.

To obtain $prob'$ for each last author of an article, we adjusted the raw probability $prob$ by the article's citation count compared to other articles of the same year.
For example, for an article published in 2010, its author has an adjusted probability of being male:
$$prob_{male}' = prob_{male} \times \frac{\sqrt{c+1}}{1/n\sum_{i} \sqrt{c_i+1}},$$
where $c$ is the number of times that article has been cited, $c_i$ is the number of times article $i$ in 2010 has been cited, and $n$ is the total number of articles published in 2010.
This weight is similar to the article's normalized citation score by year [@doi:10.1016/j.joi.2010.08.001;@doi:10.1007/s11192-019-03140-w], assuming that all the articles belong in one field (i.e., computational biology).
We applied the square root transformation to account for the skewed distribution of citation counts and the addition of 1 to avoid dropping articles with no citation.

To reiterate, we only consider the prediction probabilities in aggregate and not as individual values for specific scientists.
Moreover, although the average of the adjusted probabilities is not exactly "proportion", we use the phrase "estimated proportion" for readability.
For example, the average of adjusted probabilities of authorships with East Asian name origin is the estimate for the proportion of citation weighted authorships with East Asian names.


## Data and Resource Availability

This manuscript was written [openly on GitHub](https://github.com/greenelab/iscb-diversity-manuscript) using Manubot [@doi:10.1371/journal.pcbi.1007128].
The Manubot HTML version is available under a Creative Commons Attribution (CC BY 4.0) License at <https://greenelab.github.io/iscb-diversity-manuscript/>.
Our analysis of authors and ISCB-associated honorees is available under CC BY 4.0 at <https://github.com/greenelab/iscb-diversity>, with source code also distributed under a BSD 3-Clause License.
Rendered Python and R notebooks from this repository are browsable at <https://greenelab.github.io/iscb-diversity/>.
Our analysis of PubMed, PubMed Central, and author names relies on the Python pubmedpy package, developed as part of this project and available under a Blue Oak Model License 1.0 at <https://github.com/dhimmel/pubmedpy> and on [PyPI](https://pypi.org/project/pubmedpy/).
Our Wikipedia name dataset is dedicated to the public domain under CC0 License at <https://github.com/greenelab/wiki-nationality-estimate>, with source code to construct the dataset available under a BSD 3-Clause License.


## References {.page_break_before}

<!-- Explicitly insert bibliography here -->
<div id="refs"></div>
